### PR TITLE
Updating README. 

### PR DESCRIPTION
**Pull request by 1ozturkbe**
I felt that this is a low hanging fruit that should be picked ASAP. Feel free to update this and comment with how you would like to structure this.

### CODE REVIEW

**Comment by 1ozturkbe** [comment]
@bqpd and @mjburton11 tagged you explicitly since you two have done the most maintaining these models.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+  * Aircraft
+    * Tail
+    * Wing
+* **Miscellanous models**
+  * Economic Order Quantity
+  * Moment of Inertia
+  * Net Present Value
+  * Raymer Weights: rule-of-thumb weight relations for aircraft design
 
 [![Build Status](https://acdl.mit.edu/csi/job/gpkit_commons_Push_Models/badge/icon)](https://acdl.mit.edu/csi/job/gpkit_commons_Push_Models/)
```
Comment (mjburton11) [inline README.md]: this should be at the top

Comment (1ozturkbe) [reply README.md]: You can make edits directly on this branch using the browser!!

Comment (bqpd) [reply README.md]: Done :p

**Comment by bqpd** [comment]
I'm good to merge this now, but it'd be even better if we had one-line descriptions for every model...

**Comment by 1ozturkbe** [comment]
I agree. I put some small blurbs, but if you could fill in the rest that would be great.

**Comment by bqpd** [comment]
I took a stab at them, you or @mjburton11 should review to make sure they're not incorrect...

**Comment by mjburton11** [comment]
Sorry I was in meetings all afternoon and missed all the action

**Pull request merged**